runtime/vm/exceptiondescribe.c

```diff
@@ -354,17 +354,17 @@ iterateStackTrace(J9VMThread * vmThread, j9object_t* exception, callback_func_t
 								 * This secondary probe is required as all ROMClasses from the SCC appear to be owned
 								 * by the bootstrap classloader.
 								 */
-								ramClass = peekClassHashTable(vmThread, vm->applicationClassLoader, J9UTF8_DATA(utfClassName), J9UTF8_LENGTH(utfClassName));
-								if (NULL != ramClass) {
-									if (romClass != ramClass->romClass) {
-										/**
-										 * It is possible this romClass is not loaded by app class loader.
-										 * There is another class loader that loads a different class with the same name,
-										 * Do not return the ramClass from app loader in this case.
-										 */
-										ramClass = NULL;
-									}
-								}
+								ramClass = peekClassHashTable(vmThread, vm->applicationClassLoader, J9UTF8_DATA(utfClassName), J9UTF8_LENGTH(utfClassName))
+							}
+						}
+						if (NULL != ramClass) {
+							if (romClass != ramClass->romClass) {
+								/**
+								 * It is possible this romClass is not loaded by bootstrap/app class loader.
+								 * There is another class loader that loads a different class with the same name,
+								 * Do not return the ramClass from bootstrap/app loader in this case.
```

Comment:
I think this comment no longer really applies as this code is in the common path.

Reply:
Removed the comment.